# 1. Use already imported python dependencies.
# 2. Fix the include path for //external:zlib.
# 3. Remove -fPIC.

diff --git a/BUILD.bazel b/BUILD.bazel
index d03ca08c52..329d4141cd 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -4,7 +4,7 @@
 
 load("@bazel_skylib//lib:selects.bzl", "selects")
 load("@rules_python//python:defs.bzl", "py_binary")
-load("@v8_python_deps//:requirements.bzl", "requirement")
+load("@configs_pip3//:requirements.bzl", "requirement")
 load(
     "@v8//:bazel/defs.bzl",
     "v8_binary",
diff --git a/bazel/defs.bzl b/bazel/defs.bzl
index 0e9559a65f..582c76bca1 100644
--- a/bazel/defs.bzl
+++ b/bazel/defs.bzl
@@ -99,7 +99,6 @@ def _default_args():
         }),
         copts = select({
             "@v8//bazel/config:is_posix": [
-                "-fPIC",
                 "-fno-strict-aliasing",
                 "-Werror",
                 "-Wextra",
diff --git a/src/snapshot/snapshot-utils.cc b/src/snapshot/snapshot-utils.cc
index 6db6698d7e..b56d31085f 100644
--- a/src/snapshot/snapshot-utils.cc
+++ b/src/snapshot/snapshot-utils.cc
@@ -5,7 +5,7 @@
 #include "src/snapshot/snapshot-utils.h"
 
 #include "src/base/sanitizer/msan.h"
-#include "third_party/zlib/zlib.h"
+#include "zlib.h"
 
 namespace v8 {
 namespace internal {
